### PR TITLE
Added Initialization Modules

### PR DESCRIPTION
**Pull request by Smit1603**
- This PR will resolve issue #3 
- Initialization folder consists of :
1.  Subtractive Clustering Code File
2. K-Means++ Clustering Code File
3. Random Initialization Method Code File

### CODE REVIEW

**Comment by shahpratham** [comment]
@Smit1603 rename `summary.md` to `README.md`

File: Initialization/summary.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+value will be choose as first cluster centre. So after selecting the first cluster centre we will try to find the second cluster
+centre by calculating the highest potential value in the remaining grid points. As grid points near the first cluster center
+will reduce its potential value, the next cluster center will be grid with many data point nearby other than first cluster
+center grid point. So this procedure of acquiring new cluster center and reducing the potential of surrounding grid
+point repeat until potential of all grid points falls below a threshold value.
+
+
+## Steps
+1. Use the equation below to calculate the potential for every pixel value of the image. \
+![Screenshot (44)](https://user-images.githubusercontent.com/109454803/193085675-026e3575-3b05-4b08-b0ed-a21c0ff86bd6.png) \
```
Comment (shahpratham) [inline Initialization/summary.md]: add images in a `assets` folder and link the path to the image here

**Comment by Dhruv454000** [comment]
@ChinmayMundane Add random initialization method in readme and from references remove gfg link keep only research paper link.

**Review by Dhruv454000 — COMMENTED** [review]
Except for above changes other things LGTM

**Comment by Dhruv454000** [comment]
LGTM!

**Pull request merged**